tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "ES2018",
     "module": "commonjs",
-    "lib": ["es2018"],
+    "lib": ["es2018", "dom"],
```

Comment:
Adding `axios` is giving a Typescript error at the moment.
I followed the advice [here](https://github.com/Azure/azure-cosmos-js/issues/395#issuecomment-523957695) to add `"dom"` to the `tsconfig.json` `lib` setting to get around it.